corehq/apps/integration/payments/services.py

```diff
@@ -60,6 +60,9 @@ def request_payment(payment_case: CommCareCase, config: MoMoConfig):
         })
     except PaymentRequestError as e:
         payment_update[PaymentProperties.PAYMENT_ERROR] = str(e)
+    except Exception:
+        # We need to know when anything goes wrong
+        payment_update[PaymentProperties.PAYMENT_ERROR] = _("Something went wrong")
```

Comment:
:+1: 

nit: I believe users are more used to seeing "Unknown error" on HQ though "Something went wrong" is good as well.